### DISCUSSION TITLE
请问，mpv的标题，如何直接取文件名，而非现在默认取【属性-详细信息-标题】

### DISCUSSION THREAD

**Discussion by xiakexing2100** [hooke007/mpv_PlayKit#440] [mpv 通用]
<img width="279" alt="Snipaste_2024-04-05_05-26-24" src="https://github.com/hooke007/MPV_lazy/assets/70386677/686022da-9faf-45e6-ab58-bf896a028867">
<img width="213" alt="Snipaste_2024-04-05_05-28-30" src="https://github.com/hooke007/MPV_lazy/assets/70386677/5db430cb-9fd5-4885-925b-39c27358428b">

**Comment by hooke007** [comment]
https://github.com/hooke007/MPV_lazy/discussions/69
